src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,34 @@
+.catalog {
+  &__wrapper {
+    display: grid;
+    grid-template-columns: repeat(4, $column-width);
+    justify-content: center;
+    gap: $cards-gap;
+
+    max-width: 944px;
+    margin: 0 auto;
+
+    @media (max-width: 1024px) {
+      grid-template-columns: repeat(3, $column-width);
+    }
+
+    @media (max-width: 768px) {
+      grid-template-columns: repeat(2, $column-width);
+    }
+
+    @media (max-width: 488px) {
+      grid-template-columns: $column-width;
+    }
```

Comment:
Usually we use mixins for particular breakpoints instead of using media query every time